binding/jms/src/test/java/io/tracee/binding/jms/TraceeMessageListenerTest.java

```diff
@@ -29,7 +28,8 @@
 	private final TraceeMessageListener unit = new TraceeMessageListener(backend);
 	private final InvocationContext invocationContext = mock(InvocationContext.class);
 	private final Message message = mock(Message.class);
-	private final Map<String,String> encodedContext = new HashMap<String, String>();
+	private final Map<String, String> encodedContext = new HashMap<String, String>();
+	private MdbLike mdbLike = spy(new MdbLike());
```

Comment:
Hmm, good idea of spy(MdbLike.class). I find it fails because the MdbLike is private class. If it is a static class, it works. I'll change this later.